[rospy.client][INFO] 2022-01-07 11:42:58,287: init_node, name[/rqt_gui_py_node_25643], pid[25643]
[xmlrpc][INFO] 2022-01-07 11:42:58,288: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2022-01-07 11:42:58,288: Started XML-RPC server [http://prajwal-IdeaPad-L340-15IRH-Gaming:45657/]
[rospy.impl.masterslave][INFO] 2022-01-07 11:42:58,288: _ready: http://prajwal-IdeaPad-L340-15IRH-Gaming:45657/
[rospy.init][INFO] 2022-01-07 11:42:58,288: ROS Slave URI: [http://prajwal-IdeaPad-L340-15IRH-Gaming:45657/]
[rospy.registration][INFO] 2022-01-07 11:42:58,288: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2022-01-07 11:42:58,288: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2022-01-07 11:42:58,388: registered with master
[rospy.rosout][INFO] 2022-01-07 11:42:58,389: initializing /rosout core topic
[rospy.rosout][INFO] 2022-01-07 11:42:58,389: connected to core topic /rosout
[rospy.simtime][INFO] 2022-01-07 11:42:58,390: initializing /clock core topic
[rospy.simtime][INFO] 2022-01-07 11:42:58,391: connected to core topic /clock
[rospy.internal][INFO] 2022-01-07 11:42:58,393: topic[/clock] adding connection to [http://prajwal-IdeaPad-L340-15IRH-Gaming:33665/], count 0
[rospy.internal][INFO] 2022-01-07 11:42:58,647: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2022-01-07 11:42:58,947: topic[/robot1/cmd_vel] adding connection to [/gazebo], count 0
[rospy.internal][ERROR] 2022-01-07 11:44:03,637: Unable to initiate TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:53891 (http://prajwal-IdeaPad-L340-15IRH-Gaming:33665/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
rospy.exceptions.TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 53891 to [127.0.0.1:42710 on socket 17]] [/rqt_gui_py_node_25643].

[rospy.internal][INFO] 2022-01-07 11:44:03,920: topic[/robot1/cmd_vel] removing connection to /gazebo
[rospy.internal][WARNING] 2022-01-07 11:44:04,639: Unknown error initiating TCP/IP socket to prajwal-IdeaPad-L340-15IRH-Gaming:53891 (http://prajwal-IdeaPad-L340-15IRH-Gaming:33665/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 560, in connect
    self.socket.connect((dest_addr, dest_port))
ConnectionRefusedError: [Errno 111] Connection refused

[rospy.internal][INFO] 2022-01-07 11:44:04,639: topic[/clock] removing connection to http://prajwal-IdeaPad-L340-15IRH-Gaming:33665/
[rospy.internal][INFO] 2022-01-07 11:44:14,564: topic impl's ref count is zero, deleting topic /grid3/droppingCompleted...
[rospy.internal][INFO] 2022-01-07 11:44:14,565: topic impl's ref count is zero, deleting topic /grid3/packageLoaded...
[rospy.internal][INFO] 2022-01-07 11:44:14,566: topic impl's ref count is zero, deleting topic /robot1/cmd_vel...
[rospy.internal][INFO] 2022-01-07 11:44:14,567: topic impl's ref count is zero, deleting topic /robot2/cmd_vel...
[rospy.internal][INFO] 2022-01-07 11:44:14,567: topic impl's ref count is zero, deleting topic /robot3/cmd_vel...
[rospy.internal][INFO] 2022-01-07 11:44:14,568: topic impl's ref count is zero, deleting topic /robot4/cmd_ve...
[rospy.core][INFO] 2022-01-07 11:44:14,598: signal_shutdown [atexit]
[rospy.internal][INFO] 2022-01-07 11:44:14,600: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2022-01-07 11:44:14,600: atexit
